### DISCUSSION TITLE
Mismatch of local and central header

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#1086] [Developers]
Time to time i get Error: Mismatch of local and central header when installing new organ packages. 
Organ is then loaded properly. What is the meaning of this error ?
Thanks.

**Comment by oleg68** [comment]
It means that the .orgue file has some inconsistency.

Could you upload this .orgue somewhere? I'll test it.

**Reply by paolo2504** [reply]
Thanks Oleg for reply. 
The problem is that it occurs rarely ; in addition, it happens only once, I mean: it doesn’t happen next times organ is reloaded. 
i'am trying to figure out when it happened recently, what actions I did to try to reproduce…
I suspect issue arises when some change is made on the .orgue file… for example changing /updating the organ file. Usually I delete old organ from Settings then Install it again as package and finally reload it. 
normally , if I forgot deleting the cache file, I get only another error saying different cache file found or something like that…
Definitely, there is no  particular .orgue file causing this error…

**Reply by willeke1234** [reply]
I also have the occasional crash or weird behavior. I have converted a sample set to an organ package. When loading and reloading this organ package, GrandOrgue sometimes crashes or starts misbehaving. I think the organ package is corrupt because I can't decompress it. Or maybe it's a bug in GrandOrgue, or both.
